Makefile

```diff
@@ -6,14 +6,25 @@ build:
 generate:
 	go generate ./...
 
+install-hooks:
+	cp -r git/hooks/* .git/hooks/
+
 log:
 	tail -f *.log
 
+openapi-spec-check-updates:
```

Comment:
This will run in CI on a schedule and fail if there's a change in the OpenAPI spec file.